runtime/lang/Make_all.mak

```diff
@@ -0,0 +1,35 @@
+#
+# Makefile with common components
+#
+
+CONVERTED = \
+		menu_chinese_gb.936.vim \
+		menu_chinese_taiwan.950.vim \
+		menu_cs_cz.iso_8859-2.vim \
+		menu_czech_czech_republic.1250.vim \
+		menu_czech_czech_republic.ascii.vim \
+		menu_hu_hu.iso_8859-2.vim \
+		menu_ja_jp.euc-jp.vim \
+		menu_japanese_japan.932.vim \
+		menu_ko_kr.euckr.vim \
+		menu_pl_pl.iso_8859-2.vim \
+		menu_polish_poland.1250.vim \
+		menu_ru_ru.cp1251.vim \
+		menu_ru_ru.koi8-r.vim \
+		menu_sk_sk.iso_8859-2.vim \
+		menu_sl_si.cp1250.vim \
+		menu_sl_si.latin2.vim \
+		menu_sr_rs.ascii.vim \
+		menu_sr_rs.iso_8859-2.vim \
+		menu_sr_rs.iso_8859-5.vim \
+		menu_tr_tr.cp1254.vim \
+		menu_tr_tr.iso_8859-9.vim \
+		menu_uk_ua.cp1251.vim \
+		menu_uk_ua.koi8-u.vim \
+
```

Comment:
can you please remove those additional empty lines? and the commented out line `menu_slovak_slovak_republic`?

Reply:
@chrisbra 
Yes, I deleted the blank lines. And the commented out line as well.
But what about the rule "menu_slovak_slovak_republic.1250.vim: menu_sl_si.utf-8.vim"? I commented it out, but is it necessary to leave it in?
I wrote to @adaext , but didn't get a reply. Is it possible that it meant something else? No matter how many times I tried, I couldn't find what it should be.

Reply:
removed the rule "menu_slovak_slovak_republic.1250.vim: menu_sl_si.utf-8.vim"